src/store/modules/users.js

```diff
@@ -144,6 +145,7 @@ const actions = {
     const systemOrg = {
       id: orgUtil.SYSTEM_ORG_ID,
       name: 'SystemSetting',
+      is_system: true,
       autoEnter: new Date().getTime()
     }
     commit('SET_CURRENT_ORG', systemOrg)
```

Comment:
No differences were identified between current code and the previous version as of 2023-DATE, so it will be easier to optimize.

Potential improvements:
- Keep `state.currentOrg` up-to-date instead of hard-coding `org`.
- Use ES6 (ES5) shorthand properties where applicable (`const`, `var`) for consistency.